diff --git a/dist/opencv.js b/dist/opencv.js
index af4111b..3ba8a69 100644
--- a/dist/opencv.js
+++ b/dist/opencv.js
@@ -22,7 +22,7 @@
     // Other shells, e.g. d8
     root.cv = factory();
   }
-}(this, function () {
+}(globalThis, function () {
   
 var cv = (function() {
   var _scriptDir = typeof document !== 'undefined' && document.currentScript ? document.currentScript.src : undefined;
@@ -44,7 +44,7 @@ else if (typeof define === 'function' && define['amd'])
   define([], () => cv);
 
   if (typeof Module === 'undefined')
-    Module = {};
+    var Module = {};
   return cv(Module);
 }));
         
\ No newline at end of file
