diff --git a/dist/opencv.js b/dist/opencv.js
index ec41b85..49c466a 100644
--- a/dist/opencv.js
+++ b/dist/opencv.js
@@ -2,4 +2,4 @@
 module.exports=cv;else if(typeof define==='function'&&define['amd'])
 define([],function(){return cv;});else if(typeof exports==='object')
 exports["cv"]=cv;if(typeof Module==='undefined')
-Module={};return cv(Module);}));
\ No newline at end of file
+var Module={};return cv(Module);}));
\ No newline at end of file
